### ISSUE TITLE
The horizontal scroll is hidden for the dynamic table

### ISSUE THREAD

**Issue by eKazim** [Autodesk/react-base-table#240]
If used `estimatedRowHeight` and `fixed={false}`, there is no scroll for overflowed column width (after column resize).
For example, i want to have resizable columns and flexible last column (`flex-grow: 1`, so that it fills the free space when the other columns are small).
Playground:
https://autodesk.github.io/react-base-table/playground#MYewdgzgLgBAJgQygg4gUzGgTkkWYC8MAFAJSEB8JA3gFAwwCWcAXDAGYIDW2AdDmDggAtrwCuY5mQA09GGATC0bTjyy8FS3u0aCAcorQy5AcwxxsK7nwFDRAIxAgANmgRgyMAPwwA5MIRXXxg2X3Y0AKDZBghQLGUYOgYGAKgACys1fnc7DTFhe2xiAHYABnIAahgAZlLomABfevtGLHTEAE9MvkQoNF5CqAB3NAxiAEYATkmAVmkYACZS0pnSeqRkYDSlMCgIbvVbETyCotX6i1isRgAHKEZwA95nPAjeCAw+sGAjNbkIhCMZxPXR9LCYKC8AFA4wMUBiXZYLocazqBBwODxCAQXjwxEdWEwdGYtDYthJZLQeJoKBPYlYnFU0ZQACCGIZhLhjCgyNUfHppJxwG5BL+yRgAC9bgBhEAWOnswW8KU3WUWQlNWgNUi0WigSCwADKcTQhBg0A6rjg7xu7gABnJQC8sGwACTUG5YEA3CCUEie72Mk0wChEABspW8fhM1LAwVC8TgvlIDQA3LQHXrwNAYCyoJttp8zRarbw4IwAG4Ohj2BDALgxkAIuAAWideDYzkYJjSUBjCA66YYQ2Y6TYSxuAA8hzA0mhu73x6UpzOu5gW3OF7TFsvp3I+pOoC3At2wGwfrtsDPHFgLFgWzhy2J9jAACwrx0uDswfuDjO6-UcwsTgxGcKAABEkAQM1MCGXMsBwAkZmWco5G0IFnGIVCGF4AIbmIXpUAwbBcCwHUcIgPAoGIYgEHmexyAIKhaI0QwQxgexWKUKNxhCGAW3GUgdSzA14DQECwMNKizWoGAeGRXxNDQXx5jwO82CktoAHlbz4FlDWlRpdTQScbmk4CEFA2BgGcBBsVzG4bhgEyvjgX0ACU3GASFZWEMzMF2RI5GgJBTSICkGEItgLKsyDkHqGIqIAIWRGLJKo+oGl1OEXHySAzQAbTkBgIuSeTQiUlTiuSe4oFcUIDCUKrxUiqCAGk0AUyqEuSEc4DHGBxmQnqGCxRgJQQex6pgKAsDENARvNKjJum2b5sW9hXEnFAvSGNg6mqhhNpMw00muMAuH2nrNXFUqGHKvwWjaNJOmalraum3wktadoBze8VCI6hSnt+jp-uSQj0HzSwaBgJ08vmXa4ug7VKEO5Ikaggr4eEMAy3azqAF1eCgEAABkQGAQI0DitBDVm3QTE5XrRwyQblkWk8TDPGBZWcPLeBZLtuZ2SF3IASRQAAJAAVRbKLaFaEjWhb0eO7bdqutWttO87LpgA7xRu5I7rkzrQg2OtC12CBwYYD6El8PMC1F23FsB82-EtrZXbtmA+oGiNOeFnm+YFoXT1F3hpQAUT0GWY-cjatp2kA9oN5OTrO3R9cNlq53RbBPMEbAYc8JiYAAHmdq3RYoLxK4Aehrn3PgoRafmcZxi7vMvZM75xkcaRiqGr-Na7b6gB+Rhom5b62oHb6rjZK9GHt8aFnD9h3QhjgIgT9j2FM3v2A7ZpY8-FMaJqm5W5tVlqlsV2+2BVzONbTrXH-V3Wc6-5IV5BUfuvPEs0waLR3n4WUCIwGHwJgpUBSJT6szYOMDm6Nr5K1fvfeWy0X4zRwdrEyqd06X2SD-bOF1-4MEAabdeApsTb25J9NkJJGHu3gRbRU2J3izWZMg-q590GP0wfgt+RCP7p3GO-X+VCM7oy5qHXKuNBYhyjhLaWctl5yEJtlGA4Aw6400rAIgCsoApT9KVQCsA1LYGLMlDovBbH4AIK4mAxidJ3kFgZHifEBLVWsfAKChVeChPSIwRkyA+j42QLogGUF3hURonRDiI8SAIAKmYlKvB5KE3YvYTJDiclEyjM4vxzihLVXCYyGk9NQrEFNlkjoHD4rL3IkZOQ8QS5YE8FY7MsBZKEXmE0xoZpqm8NCtVeIUAxDghIOjSuFB0YMErvYMQ+ZwDLOSAYrs9YCDUHLlQU2j9xkfCgHUvoDStknJACYEw00ACEpyon9FJnc1wi1H7amuY0LKj8GBLP+ckGWtz7mmhxpALZTc1kbLAIC-5qz1mkzAD8nZjA9kHLScc96aQInvFqS8q5QLH5RRgAVSGxEcCkx6WsGAoSYkIEJp8lq3ziW-K2fCoFrCmB9GEDNEA-KbhQsbjC5FnKWqVxlkrH5OhJxoDgPszgzgPh-OJRCiA+yfkMAeU83FkTQok1Ba4KM4z1XvF2UYOob5yCvz1biZRkK2WqqBaSe4qR5XuTTlLecPYoD7NfKUZ1-zCL7MIkGx+TT9lNPDS1AxDrjH7PGXG-mRiqIxoYI3cVKzM3VXaVlLKQA

In css I see that the scrollbar is hidden by the style:
```
.BaseTable--dynamic .BaseTable__row {
    overflow: hidden;
    align-items: stretch;
}
```

Why is this done so? In this case, it looks like an error. Maybe there is some workaround here?

**Comment by nihgwu** [comment]
there is nothing to do with dynamic table, if you use fixed row height, there should be no horizontal scroll either if you set `fixed={false}`(which is default), in your case it could scroll horizontally only because you set `flexShrink={false}` so the width for each column is the minimum width, so the total width exceeds the table's width which should be avoided, you can see the hovering effect won't cover the whole row if you remove the `estimatedRowHeight={40}`

**Comment by nihgwu** [comment]
a fixed table means the column width is exactly same as `column.width`, and the table could be scrolled horizontally if the total width exceeds the table's width, a flexible table(fixed=false) means the column's width would shrink or grow by the flex rule, if the total width exceeds the table's width, some of the columns would be shrank, but in your case you set all the columns not shrinkable...

**Comment by eKazim** [comment]
Yes, I've already fixed the hovering effect by redefining styles and changing the cell background instead of the row's background. In another case, I have already encountered this.

I don't quite understand what difference the table is fixed or not, because it is a generally accepted rule that if the content does not fit the width of the container, then a scroll should appear, if there is an appropriate css property overflow. And this rule should also be relevant for flex containers.

Maybe there are some other restrictions in addition to the styling of row?
What could potentially happen if I redefine the style for example this way:
```
.myTable .BaseTable--dynamic .BaseTable__row {
    overflow: initial;
}
```
